cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -99,6 +145,9 @@ func NewServer(configGetter server.KubernetesConfigGetter, globalPackagingCluste
 		log.Fatalf("%s", err)
 	}
 
+	versionsInSummary := parsePluginConfig(pluginConfigPath)
+	log.Infof("NewServer: versionsInSummary %v\n", versionsInSummary)
```

Comment:
This looks like debugging rather than something we actually want to log at the info level?